javascript/javascript1/week3/note_taking_app.js

```diff
@@ -0,0 +1,62 @@
+// SAVE A NOTE
+const notepad = [];
+function saveNote(notes, content, id) {
+    const draft = {};
+    draft["content"] = content;
+    draft.id = id;
+    notes.push(draft);
+    
+}
+saveNote(notepad, "Pick up groceries", 1);
+saveNote(notepad, "Do laundry", 2);
+console.log(notepad);
+
+
+// GET A NOTE 
+function getNote (notes, id) {
+    for (let i = 0; i<notes.length; i++) {
+        if (notes[i].id === id) {
+            return notes[i];
```

Comment:
Very nice! Just wanted to add an extra example for you:
It is possible to use the array built in functions:
```
const note = notes.find((noteItem) => noteItem.id === id);
if (note === undefined) {
  return "ERROR";
}

return note;
```

Reply:
Could you explain what do you mean with "array built in functions"? If you also could break down this code step by step, it would be incredibly helpful 🙏🏻🙏🏻🙏🏻